src/projective/affine.jl

```diff
@@ -178,24 +178,45 @@ end
 Transform `P` so that is applied around the center of `bounds`
 instead of the origin
 """
-function centered(P, bounds::Bounds{2})
+function centered(P, bounds::Bounds{N}) where N
     upperleft = minimum.(bounds.rs)
     bottomright = maximum.(bounds.rs)
 
-    midpoint = SVector{2, Float32}((bottomright .- upperleft) ./ 2) .+ SVector{2, Float32}(.5, .5)
+    midpoint = SVector{N, Float32}((bottomright .- upperleft) ./ 2) .+ .5f0
     return recenter(P, midpoint)
 end
 
-
-FlipX() = Reflect(180)
-FlipY() = Reflect(90)
-
 function reflectionmatrix(r)
     A = SMatrix{2, 2, Float32}(cos(2r), sin(2r), sin(2r), -cos(2r))
     return round.(A; digits = 12)
 end
 
 
+"""
+    FlipDim{N}()
```

Comment:
Think this should be
```suggestion
    FlipDim{N}(dim)
```

Reply:
fixed